UPGRADE-6.8.md

```diff
@@ -58,6 +58,19 @@ It allows filtering of `RuleIdAware` objects in either arrays or collections.
 
 # Administration
 
+## Removed admin notification entity + related classes
+
+You should update your code to reference the new classes:
```

Comment:
we should mention, that the old classes are removed and state those as their successors

Reply:
done